[CI] FrequentItemSetsAggregatorTests testMixedSingleValues failing

This reproduced locally for me.

**Build scan:**
https://gradle-enterprise.elastic.co/s/dtzasztrsdj4u/tests/:x-pack:plugin:ml:test/org.elasticsearch.xpack.ml.aggs.frequentitemsets.FrequentItemSetsAggregatorTests/testMixedSingleValues

**Reproduction line:**
```
./gradlew ':x-pack:plugin:ml:test' --tests "org.elasticsearch.xpack.ml.aggs.frequentitemsets.FrequentItemSetsAggregatorTests.testMixedSingleValues" -Dtests.seed=BBF17791F58C24A4 -Dtests.locale=no-NO -Dtests.timezone=Asia/Pontianak -Druntime.java=17
```

**Applicable branches:**
main

**Reproduces locally?:**
Yes

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.xpack.ml.aggs.frequentitemsets.FrequentItemSetsAggregatorTests&tests.test=testMixedSingleValues

**Failure excerpt:**
```
java.lang.AssertionError: number of results do not match, expected: [{"key": {"kVal-1": {host-1}, "kVal-3": {type-1}}, "doc_count": 5, "support": 0.5}, {"key": {"fVal": {4.1}, "kVal-1": {host-1}, "kVal-2": {client-1}, "kVal-3": {type-1}}, "doc_count": 2, "support": 0.2}, {"key": {"ipVal": {192.168.0.1}, "fVal": {4.1}, "iVal": {2}}, "doc_count": 2, "support": 0.2}, {"key": {"ipVal": {192.168.0.5}, "kVal-1": {host-1}, "kVal-3": {type-1}}, "doc_count": 2, "support": 0.2}, {"key": {"kVal-1": {host-1}, "kVal-2": {client-2}, "kVal-3": {type-1}}, "doc_count": 2, "support": 0.2}, {"key": {"ipVal": {192.168.0.5}, "kVal-2": {client-2}}, "doc_count": 2, "support": 0.2}, {"key": {"kVal-2": {client-2}, "kVal-3": {type-3}}, "doc_count": 2, "support": 0.2}] got [{"key": {"kVal-1": {host-1}, "kVal-3": {type-1}}, "doc_count": 5, "support": 0.5}, {"key": {"fVal": {4}, "kVal-1": {host-1}, "kVal-2": {client-1}, "kVal-3": {type-1}}, "doc_count": 2, "support": 0.2}, {"key": {"kVal-1": {host-1}, "kVal-2": {client-2}, "kVal-3": {type-1}}, "doc_count": 2, "support": 0.2}, {"key": {"ipVal": {192.168.0.5}, "kVal-1": {host-1}, "kVal-3": {type-1}}, "doc_count": 2, "support": 0.2}, {"key": {"ipVal": {192.168.0.1}, "fVal": {4}, "iVal": {2}}, "doc_count": 2, "support": 0.2}, {"key": {"ipVal": {192.168.0.5}, "kVal-2": {client-2}}, "doc_count": 2, "support": 0.2}, {"key": {"fVal": {5}, "kVal-2": {client-2}}, "doc_count": 2, "support": 0.2}, {"key": {"kVal-2": {client-2}, "kVal-3": {type-3}}, "doc_count": 2, "support": 0.2}] parameters: minumum_support: 0.13954488727614375 minimum_set_size: 2 size: 82 expected:<7> but was:<8>

  at __randomizedtesting.SeedInfo.seed([BBF17791F58C24A4:7C505EA2243001F0]:0)
  at org.junit.Assert.fail(Assert.java:88)
  at org.junit.Assert.failNotEquals(Assert.java:834)
  at org.junit.Assert.assertEquals(Assert.java:645)
  at org.elasticsearch.xpack.ml.aggs.frequentitemsets.FrequentItemSetsAggregatorTests.assertResults(FrequentItemSetsAggregatorTests.java:796)
  at org.elasticsearch.xpack.ml.aggs.frequentitemsets.FrequentItemSetsAggregatorTests.lambda$testMixedSingleValues$3(FrequentItemSetsAggregatorTests.java:446)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.testCase(AggregatorTestCase.java:653)
  at org.elasticsearch.xpack.ml.aggs.frequentitemsets.FrequentItemSetsAggregatorTests.testMixedSingleValues(FrequentItemSetsAggregatorTests.java:343)
  at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(NativeMethodAccessorImpl.java:-2)
  at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
  at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:568)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:44)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:833)

```

--- a/x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/aggs/frequentitemsets/FrequentItemSetsAggregatorTests.java
+++ b/x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/aggs/frequentitemsets/FrequentItemSetsAggregatorTests.java
@@ -231,7 +231,6 @@ public class FrequentItemSetsAggregatorTests extends AggregatorTestCase {
         }, new AggTestConfig(builder, keywordType).withQuery(query));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/92578")
     public void testMixedSingleValues() throws IOException {
         List<MultiValuesSourceFieldConfig> fields = new ArrayList<>();
 
@@ -326,9 +325,9 @@ public class FrequentItemSetsAggregatorTests extends AggregatorTestCase {
                 0.2
             ),
             new FrequentItemSet(Map.of(IP_FIELD, List.of("192.168.0.1"), FLOAT_FIELD, List.of(4.1f), INT_FIELD, List.of(2)), 2, 0.2),
-            new FrequentItemSet(Map.of(FLOAT_FIELD, List.of(5), KEYWORD_FIELD1, List.of("host-2")), 2, 0.2),
+            new FrequentItemSet(Map.of(FLOAT_FIELD, List.of(5.0f), KEYWORD_FIELD1, List.of("host-2")), 2, 0.2),
             new FrequentItemSet(Map.of(INT_FIELD, List.of(5), KEYWORD_FIELD1, List.of("host-2")), 2, 0.2),
-            new FrequentItemSet(Map.of(FLOAT_FIELD, List.of(5), KEYWORD_FIELD2, List.of("client-2")), 2, 0.2),
+            new FrequentItemSet(Map.of(FLOAT_FIELD, List.of(5.0f), KEYWORD_FIELD2, List.of("client-2")), 2, 0.2),
             new FrequentItemSet(Map.of(IP_FIELD, List.of("192.168.0.5"), KEYWORD_FIELD2, List.of("client-2")), 2, 0.2)
         );
 
@@ -792,7 +791,12 @@ public class FrequentItemSetsAggregatorTests extends AggregatorTestCase {
             + " minimum_set_size: "
             + minimumSetSize
             + " size: "
-            + size;
+            + size
+            + " string exclude: ["
+            + stringExclude
+            + "] int exclude: ["
+            + intExclude
+            + "]";
 
         assertEquals(
             "number of results do not match, " + setsAssertMessage,

